boards/m4a-wrover/include/periph_conf.h

```diff
@@ -0,0 +1,219 @@
+ * pwm_init, the GPIOs declared for this device can be used for other
+ * purposes.
+ *
+ * @note As long as the according PWM device is not initialized with
+ * the `pwm_init`, the GPIOs declared for this device can be used
+ * for other purposes.
+ *
+ * @{
+ */
+#ifndef PWM0_GPIOS
+#if !MODULE_ESP32_WROVER_KIT_CAMERA || DOXYGEN
+#define PWM0_GPIOS  { GPIO0, GPIO4 } /**< only available when camera is not connected */
+#else
+#define PWM0_GPIOS  { }
+#endif
+#endif
+
+/** PWM_DEV(1) is not used */
+#ifndef PWM1_GPIOS
+#define PWM1_GPIOS  { }
+#endif
+/** @} */
+
+/**
+ * @name    SPI configuration
+ *
+ * SPI configuration depends on configured/connected components.
+ *
+ * HSPI is always available and therefore used as SPI_DEV(0)
+ * VSPI is only available when the camera is not plugged.
+ *
+ * @{
+ */
+
+#ifndef SPI0_CTRL
+#define SPI0_CTRL   HSPI    /**< HSPI is configured as SPI_DEV(0) */
+#endif
+
+#ifndef SPI0_SCK
+#define SPI0_SCK    GPIO14  /**< SD Card SCL */
+#endif
+#ifndef SPI0_MOSI
+#define SPI0_MOSI   GPIO15  /**< SD Card MOSI */
+#endif
+#ifndef SPI0_MISO
+#define SPI0_MISO   GPIO2   /**< SD Card MISO */
+#endif
+#ifndef SPI0_CS0
+#define SPI0_CS0    GPIO13  /**< SD Card CS */
+#endif
+
+#if !MODULE_ESP32_WROVER_KIT_CAMERA || DOXYGEN
+/**
+ * When the camera is not connected, VSPI is configured and becomes SPI_DEV(1).
+ *
+ * @note The GPIOs listed in the configuration are first initialized as SPI
+ * signals when the corresponding SPI interface is used for the first time
+ * by either calling the `spi_init_cs` function or the `spi_acquire`
+ * function. That is, they are not allocated as SPI signals before and can
+ * be used for other purposes as long as the SPI interface is not used.
+ *
+ */
+#ifndef SPI1_CTRL
+#define SPI1_CTRL   VSPI
+#endif
+
+#ifndef SPI1_SCK
+#define SPI1_SCK    GPIO19  /**< VSPI SCK used as LCD SCL, can be used to connect peripherals */
+#endif
+#ifndef SPI1_MOSI
+#define SPI1_MOSI   GPIO23  /**< VSPI MOSI used as LCD MOSI, can be used to connect peripherals */
+#endif
+#ifndef SPI1_MISO
+#define SPI1_MISO   GPIO25  /**< VSPI MISO used as LCD MISO, can be used to connect peripherals */
+#endif
+#ifndef SPI1_CS0
+#define SPI1_CS0    GPIO22  /**< VSPI CS0 used as LCD CS, can be used to connect peripherals */
+#endif
+#endif /* !MODULE_ESP32_WROVER_KIT_CAMERA || DOXYGEN */
+/** @} */
+
+/**
+ * @name   UART configuration
+ *
+ * ESP32 provides 3 UART interfaces at maximum:
+ *
+ * UART_DEV(0) uses fixed standard configuration.<br>
+ * UART_DEV(1) is not available.<br>
+ * UART_DEV(2) is not available.<br>
+ * @{
+ */
+#define UART0_TXD   GPIO1  /**< direct I/O pin for UART_DEV(0) TxD, can't be changed */
+#define UART0_RXD   GPIO3  /**< direct I/O pin for UART_DEV(0) RxD, can't be changed */
+
+#define UART1_TXD   GPIO21  /**< direct I/O pin for UART_DEV(0) TxD, can't be changed */
+#define UART1_RXD   GPIO22  /**< direct I/O pin for UART_DEV(0) RxD, can't be changed */
```

Comment:
s it compiling correctly the documentation?

Reply:
![Screen Shot 2022-01-12 at 3 42 31 PM](https://user-images.githubusercontent.com/37220428/149210389-86626a74-9f82-4901-bfd8-75edbc0e6a31.png)

Yes, compiling correctly

Reply:
:+1: